consensus/state.go

```diff
@@ -31,6 +40,7 @@ func (consensus *Consensus) startGettingNeighborConsensusState() {
 			if localConsensusHeight == 0 || localConsensusHeight+1 < majorityConsensusHeight {
 				if majorityConsensusHeight+1 > localLedgerHeight {
 					consensus.setNextConsensusHeight(majorityConsensusHeight + 1)
+					consensus.localNode.SetMinVerifiableHeight(majorityConsensusHeight + 1 + por.SigChainMiningHeightOffset)
```

Comment:
Could we make sure init node.minVerifiableHeight at here earlier than voting start?
As node.minVerifiableHeight uninitialized, it just only `0 + 1 + SigChainMiningHeightOffset = 5`

Reply:
It is initialized at the beginning of startGettingNeighborConsensusState()